README.md

```diff
@@ -116,13 +116,14 @@ The directory `/etc/cni/net.d` is the default location in which the scripts will
 Next, build the plugins:
 
 ```bash
+$ cd $GOPATH/src/github.com/containernetworking/plugins
 $ ./build.sh
 ```
 
 Finally, execute a command (`ifconfig` in this example) in a private network namespace that has joined the `mynet` network:
 
 ```bash
-$ CNI_PATH=`pwd`/bin
+$ CNI_PATH=$GOPATH/src/github.com/containernetworking/plugins/bin
 $ cd scripts
```

Comment:
I think this needs to change to
```
cd $GOPATH/src/github.com/containernetworking/cni/scripts
```

The same as what you already did below for the Docker instructions.